### PR TITLE
Update game over message

### PR DESCRIPTION
**Pull request by fffriday94**
K

### CODE REVIEW

**Review by fffriday94 — COMMENTED** [review]
I like.

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (engiecat) [inline index.html:50]: H E H

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (fffriday94) [inline index.html:49]: I hate this course.

**Review by engiecat — COMMENTED** [review]
test test test

**Review by fffriday94 — COMMENTED** [review]
k

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (fffriday94) [inline index.html]:
```suggestion
<h2 hidden>Game over 🧑‍💻 🤖!</h2>
```

**Review by fffriday94 — COMMENTED** [review]
k

File: index.html [hunk 4]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
```
Comment (engiecat) [inline index.html:51]:
```suggestion
this is garbage
```

File: index.html (hunk 2 above)
Comment (engiecat) [inline index.html:49]:
```suggestion
this is garbage
```

**Review by engiecat — COMMENTED** [review]
reject this review

**Pull request merged**